patterns/contact-info-locations.php

```diff
@@ -0,0 +1,93 @@
+	<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--primary);border-top-width:1px;margin-top:var(--wp--preset--spacing--50);padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)">
+		<!-- wp:group {"style":{"layout":{"columnSpan":1,"rowSpan":1}},"layout":{"type":"flex","orientation":"vertical"}} -->
+		<div class="wp-block-group">
+			<!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"small"} -->
+			<h3 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">Social media</h3>
+			<!-- /wp:heading -->
+			<!-- wp:navigation {"overlayMenu":"never","style":{"spacing":{"blockGap":"0"}},"fontSize":"small","layout":{"type":"flex","orientation":"vertical"},"ariaLabel":"<?php esc_attr_e( 'Social', 'twentytwentyfive' ); ?>"} -->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'X/Twitter', 'twentytwentyfive' ); ?>","url":"#"} /-->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'Instagram', 'twentytwentyfive' ); ?>","url":"#"} /-->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'Facebook', 'twentytwentyfive' ); ?>","url":"#"} /-->
+				<!-- wp:navigation-link {"label":"<?php esc_html_e( 'TikTok', 'twentytwentyfive' ); ?>","url":"#"} /-->
+			<!-- /wp:navigation -->
+			<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"small"} -->
+			<h2 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">Email:</h2>
+			<!-- /wp:heading -->
+			<!-- wp:paragraph {"fontSize":"small"} -->
+			<p class="has-small-font-size">Example@example.com</p>
+			<!-- /wp:paragraph -->
+		</div>
+		<!-- /wp:group -->
+
+		<!-- wp:group {"layout":{"type":"grid","minimumColumnWidth":null,"columnCount":"2"}} -->
+		<div class="wp-block-group">
+			<!-- wp:group {"style":{"layout":{"columnSpan":1,"rowSpan":1}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"small"} -->
+				<h3 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">New York</h3>
+				<!-- /wp:heading -->
+				<!-- wp:paragraph {"fontSize":"small"} -->
+				<p class="has-small-font-size">123 Example St.Manhattan, NY 10300 United States</p>
+				<!-- /wp:paragraph -->
+			</div>
+			<!-- /wp:group -->
+
+			<!-- wp:group {"style":{"layout":{"columnSpan":1,"rowSpan":1}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"small"} -->
+				<h3 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">New York</h3>
```

Comment:
```suggestion
				<h3 class="wp-block-heading has-small-font-size" style="font-style:normal;font-weight:700">San Diego</h3>
```

Reply:
I did not include this on purpose because It would only increase the number of placeholder texts that needs to be translated, and has no other purpose since it needs to be replaced by the user.

Reply:
I agree with you @carolinan but having same content on all placed doesn't look good. :)

Reply:
I'd tend to agree with kafleg, the added diversity serves a role in inspiring the user with what they can do. I'd suggest it's worth the extra translation effort.